### ISSUE TITLE
NN paper mistake report

### ISSUE THREAD

**Issue by DutyCat** [FederatedAI/FATE#2959]
**Describe the bug**
in 《Deep Neural Network for Collaborative Machine Learning with Additively Homomorphic Encryption》paper，there are two oblivious mistakes。
1）at algorithm 1,original description is “ai ← σ(W2x)”，it should be "ai ← σ(W1x)".
2）at section3.4，the learning rate η should be outing of the bracket。

**Comment by dylan-fan** [comment]
thanks for your feedback.

**Comment by DutyCat** [comment]
okk
